drivers/pinmux/stm32/pinmux_stm32l1x.h

```diff
@@ -214,4 +214,9 @@
 #define STM32L1X_PINMUX_FUNC_PF10_ADC1_IN31  \
 	STM32_MODER_ANALOG_MODE
 
+#define STM32L1X_PINMUX_FUNC_PA4_DAC_OUT1 \
```

Comment:
pinmux should be placed in order.

Reply:
Looks like there are inconsistencies in pinmux ordering between STM32 series. I'll move these lines.

Reply:
Done.